src/constants.ts

```diff
@@ -12,6 +12,7 @@ export const CAMPAIGN_IDS_KEY = '__cmpns'
 export const EVENT_TIMERS_KEY = '__timers'
 export const AUTOCAPTURE_DISABLED_SERVER_SIDE = '$autocapture_disabled_server_side'
 export const HEATMAPS_ENABLED_SERVER_SIDE = '$heatmaps_enabled_server_side'
+export const WEB_VITALS_ENABLED_SERVER_SIDE = '$wvess'
```

Comment:
Is there a reason to truncate this? Feels less explicit that the other flags

Reply:
only bundle size... it's not for humans so 🤷